docs/tasksDir/pod-security-admission-with-kubewarden.md

```diff
@@ -150,20 +155,38 @@ spec:
 EOF
 ```
 
-This time the Pod Security Admission will allow the Pod to be created, but the Pod creation will still fail.
+</details>
 
-```console
+<!--TODO:Better technical description needed-->
+
+<!--
+
+Current sentence is
+
+Now PSA allows the pod to be created, but the Pod creation will still fail.
+
+Is this better?
+
+Now the PSA process passes pod creation but the Kubewarden process of pod creation fails.
```

Comment:
that's incorrect, as I understand it, Kubewarden is not yet in the mix.
I think we are just showcasing a shortcoming of PSA behaviour, that then we fix with a Kubewarden mutating policy. So I suppose the current phrase is okeish.

Reply:
Ok, think I have it now.